test/lit/exec/table.grow.wast

```diff
@@ -0,0 +1,18 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --output=fuzz-exec and should not be edited.
+
+;; RUN: wasm-opt %s -all --fuzz-exec-before -q -o /dev/null 2>&1 | filecheck %s
+
+(module
+ (table $0 1 funcref)
+
+ ;; CHECK:      [fuzz-exec] calling test
+ ;; CHECK-NEXT: [fuzz-exec] note result: test => -1
+ (func $test (export "test") (result i32)
```

Comment:
Can we test that growing right up to the limit works?

Reply:
Makes sense, added.